docs/proposals/cloudcore-ha-design.md

```diff
@@ -0,0 +1,162 @@
+---
+title: CloudCore HA Design
+authors:
+  - "@GsssC"
+  - "@Poor12"
+approvers:
+  - "@kevin-wangzefeng"
+  - "@fisherxu"
+  - "@kadisi"
+
+creation-date: 2020-03-28
+last-updated: 2020-05-09
+status: alpha implemented
```

Comment:
We can delete the alpha  status here, only leave `Implemented`

Reply:
done : )